dts/msm8939-alcatel-idol3.dts

```diff
@@ -0,0 +1,13 @@
+// SPDX-License-Identifier: GPL-2.0-only
+
+/dts-v1/;
+
+/include/ "msm8916.dtsi"
+
+/ {
+	qcom,msm-id = <239 0>;
+	qcom,board-id = <8 0>;
+
+	model = "Alcatel OneTouch Idol 3 (5.5)";
+	compatible = "alcatel,idol3", "qcom,msm8939", "lk2nd,device";
```

Comment:
Mentioned that in the chat already: I think we should match cmdline here like for some other devices since this is just `msm8939-mtp` actually.